server/webhook/merge_request.go

```diff
@@ -23,6 +23,11 @@ func (w *webhook) handleDMMergeRequest(event *gitlab.MergeEvent) ([]*HandleWebho
 	authorGitlabUsername := w.gitlabRetreiver.GetUsernameByID(event.ObjectAttributes.AuthorID)
 	senderGitlabUsername := event.User.Username
 
+	toUsers := []string{authorGitlabUsername}
+	for _, assigneeID := range event.ObjectAttributes.AssigneeIDs {
+		toUsers = append(toUsers, w.gitlabRetreiver.GetUsernameByID(assigneeID))
+	}
```

Comment:
It seems this will make it so the author and all assignees will receive the message "requested your review" etc. We don't want to send the MR author a message saying that they requested their own review on their own MR. It seems we were already doing this though? I don't know. This code in general is kind of confusing.

Also we'll want to make sure we don't duplicate the author in the array if they are also an assignee.

What exactly is the meaning of `event.ObjectAttributes.AssigneeIDs`? Is it "all users currently assigned to this merge request", or "all users relevant to this event"?

Reply:
@mickmister if an author assigns a PR to himself / herself, he / she will not get the DM "...requested your review...". This message is sent to all other assignees except for the author.

Reply:
> This message is sent to all other assignees except for the author.

I think you may mean "all other assignees except for the issue updater" right?